docs/tutorials/ad-hoc-simulation/.pages

```diff
@@ -2,4 +2,3 @@ nav:
   - index.md
   - planning-simulation.md
   - rosbag-replay-simulation.md
-  - digital-twin-simulation.md
```

Comment:
@kenji-miyake I had assumed that since the AWSIM was going to be the official AWF E2E simulator, the tutorial in this PR would replace that file, but Wojciech will be able to explain better.

@WJaworskiRobotec Could you provide some more detail about why the `digital-twin-simulation.md` file was removed please?

Reply:
I know it was moved to a new place, but when we move a document, we also have to update the parent page.
![image](https://user-images.githubusercontent.com/31987104/182066305-9a4e6f3e-0d4c-4153-9ce3-714bc59ab648.png)